install/linux/vendor-pymupdf.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import sys
+from pathlib import Path
+
+DZ_VENDOR_DIR = Path("./dangerzone/vendor")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--dest",
+        default=DZ_VENDOR_DIR,
+        help="The destination directory for the vendored packages (default: ./dangerzone/vendor)",
+    )
+    args = parser.parse_args()
+
+    print(">>> Getting PyMuPDF deps as requirements.txt", file=sys.stderr)
+    cmd = ["poetry", "export", "--only", "container"]
+    container_requirements_txt = subprocess.check_output(cmd)
+
+    print(f">>> Vendoring PyMuPDF under '{args.dest}'", file=sys.stderr)
```

Comment:
I've just read https://pip.pypa.io/en/latest/user_guide/#using-pip-from-your-program which explains why we shouldn't use pip internal library, maybe it's worth dropping a line about this here?

Reply:
Sure thing, added in 3892dec4.